standard/unsafe-code.md

```diff
@@ -99,8 +99,8 @@ A *pointer_type* is written as an *unmanaged_type* ([§9.8](types.md#98-unmanage
 
 ```ANTLR
 pointer_type
-    : unmanaged_type '*'
-    | 'void' '*'
+    : value_type '*'+
```

Comment:
This change is a consequence of changing `unmanaged_type`, it needed to be changed to allow pointers to pointers. The version included does this with the `+` operator, while this is used elsewhere in the grammar it is usually applied to non-terminals and here it is to a literal (`'*'+`). An alternative would be to make the rule left-recursive, which ANTLR can handle, by dropping the `+`'s and adding an alternative `pointer_type '*'`. The version here produces a "flat-wide" parse/node, the alternative produces a "tree" parse/of nodes. I choose "flat-wide", but one could argue `'*'+` is a little hard to read... Express your preference!

Reply:
I think putting parentheses, as in `('*')+`, would be enough to make it more readable.

Reply:
@MadsTorgersen –  That's harmless enough ;-) Done.